.circleci/config.yml

```diff
@@ -355,6 +382,206 @@ jobs:
       - store_artifacts:
           name: "Store logs"
           path: /tmp/e2e
+  e2e_testing:
+    parameters:
+      k8sVersion:
+        description: version of k3s to use or "kind" and "kindIpv6"
+        type: string
+        default: << pipeline.parameters.last_k8s_version >>
+      parallelism:
+        description: level of parallelization
+        type: integer
+        default: 1
+      target:
+        description: makefile target without test/e2e prefix
+        type: string
+        default: ""
+      arch:
+        description: The golang arch
+        type: string
+        default: amd64
+      cniNetworkPlugin:
+        description: The CNI networking plugin to use [flannel | calico]
+        type: string
+        default: flannel
+      legacyKDS:
+        description: if should run tests with new implementation of KDS
+        type: boolean
+        default: false
```

Comment:
I don't think you should have defaults for these parameters. They should just always be set whenever this is triggered